lib/google-analytics.js

```diff
@@ -52,30 +44,35 @@ module.exports = (function () {
 
     GoogleAnalytics.getCid(function (cid) {
       _.extend(params, { cid: cid }, GoogleAnalytics.defaultParams());
-      GoogleAnalytics.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
+      this.request('https://www.google-analytics.com/collect?' + querystring.stringify(params));
```

Comment:
Google analytics are left out of specs as of now. Not sure if it make sense to test it or not. If it breaks it wont affect user experience anyway